sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/Command.scala

```diff
@@ -27,4 +27,8 @@ import org.apache.spark.sql.catalyst.expressions.Attribute
 trait Command extends LogicalPlan {
   override def output: Seq[Attribute] = Seq.empty
   override def children: Seq[LogicalPlan] = Seq.empty
+  // Commands are eagerly executed. They will be converted to LocalRelation after the DataFrame
+  // is created. That said, the statistics of a command is useless. Here we just return a dummy
+  // statistics to avoid unnecessary statistics calculation of command's children.
+  override def stats: Statistics = Statistics(Long.MaxValue)
```

Comment:
nit: To tell a difference between this dummy stat and the default table stat (e.g., Statistics(`defaultSizeInBytes`=Long.MaxValue)), how about defining a dymmy stat as a singleton in `Statistics` like `Statistics.dymmyStat`?

Reply:
+1 for @maropu 's comment.

Reply:
yea, +1 for the comment.